docs/sources/troubleshooting/troubleshoot-queries.md

```diff
@@ -22,3 +22,10 @@ like a bug or visualization issue in Grafana, it is almost always a problem with
 the data source response. Start by inspecting your panel query and response.
 
 For more information, refer to [Inspect a panel]({{< relref "../panels/inspect-panel.md" >}}).
+
+## My query is slow
+
+How many data points is your query returning? Queries returning lots of data points will be slow. Try this:
```

Comment:
Maybe "How many data points are your queries returning?" Since the following sentence or plural. Or flip the other side "A query returning...".

Reply:
I think it's written in singular since the section topic is the same.